src/constructs/cloudwatch/ec2-alarms.ts

```diff
@@ -43,7 +45,7 @@ export class Gu5xxPercentageAlarm extends GuAlarm {
       comparisonOperator: ComparisonOperator.GREATER_THAN_THRESHOLD,
       alarmName: props.alarmName ?? defaultAlarmName,
       alarmDescription: props.alarmDescription ?? defaultDescription,
-      evaluationPeriods: props.numberOfFiveMinutePeriodsToEvaluate ?? 1,
+      evaluationPeriods: props.numberOfMinutesAboveThresholdBeforeAlarm ?? 1,
```

Comment:
@sihil - I know you had some concerns about noisy alarms, so it'd be interesting to hear your take on this:

>  By default, we will now move into an alarm state when errors have been above the user-specified threshold for 1 minute.